src/AttentiveServiceProvider.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Astrogoat\Attentive;
+
+use Astrogoat\Attentive\Settings\AttentiveSettings;
+use Helix\Lego\Apps\App;
+use Helix\Lego\LegoManager;
+use Spatie\LaravelPackageTools\Package;
+use Spatie\LaravelPackageTools\PackageServiceProvider;
+
+class AttentiveServiceProvider extends PackageServiceProvider
+{
+    public function registerApp(LegoManager $stratum)
+    {
+        $stratum->registerApp(function (App $app) {
+            return $app
+                ->name('attentive')
+                ->settings(AttentiveSettings::class)
+                ->migrations([
+                    __DIR__ . '/../database/migrations',
+                    __DIR__ . '/../database/migrations/settings',
+                ])
+                ->backendRoutes(__DIR__.'/../routes/backend.php')
+                ->frontendRoutes(__DIR__.'/../routes/frontend.php');
```

Comment:
You can remove this two since we're not going to use them.

```suggestion
                ]);
```